data/human/culture conversations.txt

```diff
@@ -103,6 +103,35 @@ mission "Trash Fire Crops"
 
 
 
+mission "Worker's Occasion"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you, but you recall how unreachable they would've been for you before your career change.`
+			choice
+				`	(Offer to pay for the man's dinner.)`
+					goto aid
+				`	(Silently pity the man.)`
+					goto pity
+				`	(Smile at the man.)`
+			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go about your business.`
+				decline
+			label aid
+			`	You enter the restaurant and catch the man's attention, offering to pay for his meal.`
+			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection, you think that perhaps your pride would've been rather heavily wounded were you in his position.`
```

Comment:
Not a big fan of telling the player what they should be feeling.
```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. As you walk away, you think about what your reaction would be if a much wealthier individual pitied you by paying for your dinner.`
```

Reply:
Mmm, maybe, but it is only "perhaps", and at this stage the player chose the dumbass option

Reply:
Lol I would've unironically chosen the dumbass option trying to be nice
I do agree that this is probably good as-is though.

Reply:
Maybe it's my own personal disposition, but that line really took me out of the conversation, especially when considering how the rest of the game is generally written.